### ISSUE TITLE
Cron file being truncated since 2.9.12

### ISSUE THREAD

**Issue by silvervest** [ansible/ansible#71368]
##### SUMMARY
Since installing 2.9.12, running multiple `cron:` tasks one after another on the same `cron_file` leads the file to be truncated for every entry, leaving only the final task entry executed

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
cron

##### ANSIBLE VERSION

```paste below
$ ansible --version
ansible 2.9.12
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/silvestron2c/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.8.2 (default, Jul 16 2020, 14:00:26) [GCC 9.3.0]
```

##### CONFIGURATION

```paste below

```

##### OS / ENVIRONMENT

Ubuntu 20.04

##### STEPS TO REPRODUCE

New playbook, multiple cron tasks all writing to the same cron_file entry

```yaml
playbook:
---
- name: run locally
  hosts: 127.0.0.1
  connection: local
  tasks:
    - cron:
        cron_file: "/etc/cron.d/test"
        user: "ubuntu"
        name: "First cron"
        job: "echo first"
    - cron:
        cron_file: "/etc/cron.d/test"
        user: "ubuntu"
        name: "Second cron"
        job: "echo second"
```

run the palybook:
```
$ sudo ansible-playbook -k -e ansible_connection=local test.yml
SSH password:

PLAY [run locally] ********************************************

TASK [Gathering Facts] ******************************************
ok: [localhost]

TASK [cron] ********************************************
changed: [localhost]

TASK [cron] ********************************************
changed: [localhost]

PLAY RECAP ********************************************
localhost                  : ok=3    changed=2    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```

after the playbook is run,
```
$ cat /etc/cron.d/test
#Ansible: Second cron
* * * * * ubuntu echo second
```

##### EXPECTED RESULTS

The cron file to have multiple job entries.

Downgrading to 2.9.11 and running the same playbook results in the expected outcome.

##### ACTUAL RESULTS

```paste below

```

**Comment by silvervest** [comment]
This may be related to https://github.com/ansible/ansible/issues/69492 as it's the only change to the cron module in a long time, and it was deployed in 2.9.12

**Comment by silvervest** [comment]
If I run the playbook again while tailing the newly created cron file in another term, tail clearly shows the file being truncated along the way:
```
$ tail -f /etc/cron.d/test
#Ansible: Second cron
* * * * * ubuntu echo second
tail: /etc/cron.d/test: file truncated
#Ansible: First cron
* * * * * ubuntu echo first
tail: /etc/cron.d/test: file truncated
#Ansible: Second cron
* * * * * ubuntu echo second
```

**Comment by Akasurde** [comment]
Fixed via https://github.com/ansible/ansible/pull/71244

**Comment by Akasurde** [comment]
@silvervest Thanks for reporting this issue. Could you please verify and let me know if this fixed via #71244

**Comment by silvervest** [comment]
Yes, that commit resolves the issue.
Is there a schedule to release this change? It's a pretty significant regression...

**Comment by Akasurde** [comment]
@silvervest Changes are merged in 2.9.12 branch. So we are expecting release in this coming days.
